src/encoded/static/scss/encoded/modules/_static-pages.scss

```diff
@@ -513,11 +521,13 @@ body[data-path="/"]>#application>#layout {
 			}
 
 			> .alert {
-				background-color: #FEE5E5;
 				position: relative;
 				padding: 17px 60px 17px 25px;
-
-				.alert-heading {
+				
+				&-danger {
+					background-color: #FEE5E5;
+				}
```

Comment:
In the future, we will need to override this stuff in bootstrap overrides. This is fine for now, though. If it works, do not change it.